### ISSUE TITLE
Configuration files for rule conversion using sigmac

### ISSUE THREAD

**Issue by Iqi-Malick** [SigmaHQ/sigma#3231]
Currently I am converting zeek sigma rules to elastalert using sigmac. Configuration file used for this is ecs-zeek-elastic-beats-implementation

Following command is used for conversion:

`sudo ./sigmac -I -t elastalert -r rules/network/zeek/ -c ecs-zeek-elastic-beats-implementation -o zeek/ -e yml`

I want to convert all the sigma rules of type network, application, compliance, linux etc using one general configuration file. Is it possible to do? If yes, then a little guideline in this regard would be appreciated.

Thank you!

**Comment by nasbench** [comment]
Hi Thanks for opening the issue and sorry for the late response. 

To my knowledge and If I understood the question correctly, this should possible in `sigmac` as the `-c/--config` accepts multiple configs at once and merges them. Just make suer that no conflict in mappings and use the correct order of configs to avoid this. (See [Sigmac Wiki](https://github.com/SigmaHQ/sigma/blob/master/tools/README.md)).

So basically choose the correct config(s) file(s) related to each rule log source you want to translate and pass them all to `sigmac` and it should work.

Just for additional feedback, I'll cc @frack113 and @thomaspatzke (Please correct me If I'm wrong)

**Comment by thomaspatzke** [comment]
This works exactly as @nasbench says. It is important that field mappings in sigmac are not logsource-aware. This can cause that the mapping for a different logsource is inadvertently applied.
